tools/tpm2_pkcs11/tpm2.py

```diff
@@ -53,6 +52,18 @@ def evictcontrol(ownerauth, ctx):
 
     def load(self, pctx, pauth, priv, pub):
 
+        if isinstance(priv, buffer):
```

Comment:
The `buffer` type doesn't exist any more in Python 3 and has been replaced by [`bytes` in `sqlite3`]( https://docs.python.org/3/library/sqlite3.html#sqlite-and-python-types). Using
```
if not isinstance(priv, str):
```
works in Python 2 and 3.